spec/03-plugins/28-grpc-gateway/01-proxy_spec.lua

```diff
@@ -1,7 +1,7 @@
 local cjson = require "cjson"
 local helpers = require "spec.helpers"
 
-for _, strategy in helpers.each_strategy() do
+for _, strategy in helpers.each_strategy({"postgres"}) do
```

Comment:
why do you choose `each_strategy({"postgres"})`?

Reply:
restored